src/database/models/activationcodes.js

```diff
@@ -0,0 +1,26 @@
+"use strict";
+const { Model } = require("sequelize");
+module.exports = (sequelize, DataTypes) => {
+  class activationCodes extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      // define association here
+      this.myAssociation = this.belongsTo(models.User);
```

Comment:
I suppose the `myAssociation` name is a leftover copy-pasta?

Reply:
Also the `associate()` method is not called anywhere, despite the comment saying it is